tests/modules/nf-scil/bundle/recognize/nextflow.config

```
@@ -0,0 +1,5 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

publishDir should be within a withName: MODULE_NAME. It prevents generating md5sum for every file within the test data rather than just those outputted from the module. Once done, you will need to rerun the create-test-yml. 
```suggestion
process {
    withName: "BUNDLE_RECOGNIZE" {
         publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
     }
}
```

---

Thanks!